agent/structs/acl.go

```diff
@@ -1337,7 +1329,7 @@ type ACLTokenBatchSetRequest struct {
 // This is particularly useful during token replication when
 // multiple tokens need to be removed from the local DCs state.
 type ACLTokenBatchDeleteRequest struct {
-	TokenIDs []string // Tokens to delete
+	TokenAccessorIDs []string // Tokens to delete
```

Comment:
I suspect that this will be a breaking change because this type is used for Raft Apply operations and the field names are serialized, so this cause older versions of Consul servers to fail to decode the newer version of this message.
cc @rboyer

Reply:
A quick search seems to imply the field names in msgpack are required to not change: https://msgpack.org/index.html

Reply:
Thanks for flagging, reverted